vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -551,14 +551,14 @@ - (void)saveBundleState:(std::unique_ptr<ads::BundleState>)state callback:(ads::
 
 #pragma mark - Logging
 
-- (void)eventLog:(const std::string &)json
+- (void)log:(const char *)file line:(const int)line severity:(const ads::LogSeverity)severity message:(const std::string &) message
 {
-  [self log:__FILE__ line:__LINE__ logLevel:ads::LogLevel::LOG_INFO]->stream() << json << std::endl;
+  [self log:__FILE__ line:__LINE__ severity:severity message:message];
```

Comment:
Both changes you made turn these log calls into recursive/cyclical function calls… I'm not sure what the intention was here

Reply:
I have updated to use the log-stream, thanks